### PR TITLE
feat: lists component and fixed light mode/dark mode

### PR DESCRIPTION
**Pull request by vanshitalavange**
Overview:
- created list component 
- updated all links for list component in all other files
- gave class names to anchor tags to access in light dark mode
- fixed functionality of light dark mode

Preview:
screenshot (6)

### CODE REVIEW

**Review by vaibhav-mahalle — APPROVED** [review]
learned something new while reviewing the pull request. liked the use of separate commit messages for a particular feature.

File: JS/light-dark-mode.js [hunk 1]
```diff
@@ -1,3 +1,8 @@
+
+if (localStorage.getItem('theme') === null && location.pathname === "/index.html") {
+  console.log('yes null')
```
Comment (anushasunil) [inline JS/light-dark-mode.js:3]: assuming this to be redundant code so you can discard

File: docs/docs-layout/docs.css [hunk 2]
```diff
@@ -1,6 +1,6 @@
 body {
-  background-color: var(--dark-theme-bg);
-  color: var(--white-color);
+  /* background-color: var(--dark-theme-bg);
```
Comment (anushasunil) [inline docs/docs-layout/docs.css:2]: can remove these commented code later on

File: JS/light-dark-mode.js (hunk 1 above)
Comment (anushasunil) [inline JS/light-dark-mode.js]: assuming this to be redundant code so you can discard

File: JS/light-dark-mode.js [hunk 3]
```diff
@@ -117,10 +146,15 @@ const checkSelectedMode = (mode) => {
...
-};
-checkSelectedMode(localStorage.mode);
+}
+btnLightMode.addEventListener("click", () => {
+  localStorage.setItem('theme', 'light')
+  changeTheme(localStorage.getItem('theme'))
+});
+
+btnDarkMode.addEventListener("click", () => {
+  localStorage.setItem('theme', 'dark')
```
Comment (anushasunil) [inline JS/light-dark-mode.js:156]: Really liked this method of using `localStorage` to keep a track of the mode selected. Got to learn something new, will surely learn more and implement, Very well done 👍

File: docs/buttons/buttons.html [hunk 4]
```diff
@@ -100,11 +103,6 @@ <h1 class="content-title">Buttons</h1>
           </p>
         </div>
 
-        <!-- <div class="content flex-column">
```
Comment (anushasunil) [inline docs/buttons/buttons.html:103]: can discard commented-out code

**Review by anushasunil — APPROVED** [review]
Code looks well structured, very small refactoring suggested which can be done later on.
You are good to go!👍

File: docs/buttons/buttons.html (hunk 4 above)
Comment (vanshitalavange) [reply docs/buttons/buttons.html:103]: it is in red, so that's already deleted

File: JS/light-dark-mode.js (hunk 1 above)
Comment (vanshitalavange) [reply JS/light-dark-mode.js]: done

**Pull request merged**